scripts/setup-test-envvars.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash -e
+
+# setup-test-envvars.sh
+#
+# A helper script that uses envchain (https://github.com/sorah/envchain) to set environment variables for tests.
+# It's required to have TFE_ADDRESS and TFE_TOKEN set, the others are optional.
+#
+type envchain >/dev/null 2>&1 || { echo >&2 "Required executable 'envchain' not found - install it via 'brew install envchain'. Exiting."; exit 1; }
+
+echo "Set environment variables (envvars) for running tests locally"
+echo " envchain will prompt you for values for 4 envvars"
+echo " TFE_ADDRESS and TFE_TOKEN are required, the others are optional, hit 'return' to skip them"
+echo ""
+echo "What workspace name do you want to use in envchain?"
+read workspace
```

Comment:
Should we name this "namespace" instead to avoid confusion with the popular "workspace" term used by terraform?